.github/tests/invalid/script/0.0.1/invalid-naming.yml

```diff
@@ -0,0 +1,13 @@
+version: 0.0.1
+
+# Shell would be accepted to indicate a single line shell command (or similar)
+login_node&_check:
```

Comment:
i take it subtest cant have special characters. Make sense to me

Reply:
correct!